_posts/2020-09-25-nodejs-010-query.adoc

```diff
@@ -0,0 +1,31 @@
+---
+layout: blog
+title: Infinispan Node.js client supports query
+permalink: /blog/:year/:month/:day/infinispan-js-client
+date: '2022-09-25T12:00:00.000-00:00'
+author: Neeraj Gartia
```

Comment:
@NeerajGartia21 please add a short bio in '_data/authors.yml' and change the author id here accordingly

Reply:
sure sir!